discord/utils.py

```diff
@@ -626,7 +634,7 @@ async def maybe_coroutine(f: MaybeAwaitableFunc[P, T], *args: P.args, **kwargs:
 
 async def async_all(gen: Iterable[Awaitable[T]], *, check: Callable[[T], bool] = _isawaitable) -> bool:
     for elem in gen:
-        if check(elem):
+        if check(elem): # type: ignore
```

Comment:
Please remove this.

Reply:
ok

Reply:
but was giving me a type error

Reply:
![image](https://user-images.githubusercontent.com/34207400/173296783-a9ef7b18-8fde-432b-a9b8-5265f00feffa.png)
Here this is the error that vscode was giving

Reply:
It was already fixed in a separate PR. Please keep PRs to a single issue.

Reply:
ohhh ok !!